tests/wsgi_test.py

```diff
@@ -1743,7 +1743,9 @@ def handler(*args):
             fd = self.connect()
             fd.sendall(req.encode())
             fd.close()
-            eventlet.sleep(0.0)
+
+            # TODO changed from 0 to make Python 3 tests pass, not sure if ok
+            eventlet.sleep(0.01)
```

Comment:
0.01 is magic number. Two `sleep(0)` also work and could be explained like "wait for complete send, wait for complete receive". It may be not actually these two events that we yield to with sleep, so it doesn't fit as comment without prior investigation, I only establish a meaningful example.